src/main/java/com/amazonaws/services/kinesis/leases/interfaces/ILeaseRenewer.java

```diff
@@ -97,4 +97,9 @@
     boolean updateLease(T lease, UUID concurrencyToken)
         throws DependencyException, InvalidStateException, ProvisionedThroughputException;
 
+    /**
+     * Shutdown any clients and thread-pools.
+     */
+    void shutdown();
```

Comment:
Could you add a default behavior or else this would become a breaking change.

```java
default void shutdown() {
    // Does nothing
}
```

Reply:
Will add.  Not sure why this interface was public, since it wasn't injectable. Would this be a breaking change if it wasn't injectable?

Reply:
Changed.